### ISSUE TITLE
A11y: missing focus on pdp for gallery

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7879]
### PHP Version

8.2

### Shopware Version

6.6.10.2

### Affected area / extension

Platform(Default)

### Actual behaviour

When tabbing through the PDP, the focus disappears at the gallery arrow button. Interaction is also not possible at this point.

The problem only occurs if there are several images in the gallery.

### Expected behaviour

The tabbing through the gallery on the PDP with the arrow buttons works. 

### How to reproduce

Enable ` ACCESSIBILITY_TWEAKS` flag and enter a PDP page with 2 or more product images.

![Image](https://github.com/user-attachments/assets/74179412-6d78-4579-9681-3eed2f697c5d)

**Comment by Phil23** [comment]
Hey @73nici , the missing focus is intended. The slider can be navigated via the keyboard via the tab key as a list of items. The arrow controls, therefore, only function as a visual element for mouse users. Jumping back and forth between arrow controls and the actual slider content does not provide a good user experience for keyboard users.
